File: src/pytra/runtime/std/typing.py
from __future__ import annotations

Any: int = 1
List: int = 1
Set: int = 1
Dict: int = 1
Tuple: int = 1
Iterable: int = 1
Sequence: int = 1
Mapping: int = 1
Optional: int = 1
Union: int = 1
Callable: int = 1
TypeAlias: int = 1


def TypeVar(name: str) -> int:
    _ = name
    return 1

File: src/runtime/cpp/pytra/std/typing.cpp
#include "runtime/cpp/py_runtime.h"


namespace pytra::std::typing {

    
    int64 Any = 1;
    
    int64 List = 1;
    
    int64 Set = 1;
    
    int64 Dict = 1;
    
    int64 Tuple = 1;
    
    int64 Iterable = 1;
    
    int64 Sequence = 1;
    
    int64 Mapping = 1;
    
    int64 Optional = 1;
    
    int64 Union = 1;
    
    int64 Callable = 1;
    
    int64 TypeAlias = 1;
    
    int64 TypeVar(const str& name) {
        str _ = name;
        return 1;
    }
    
}  // namespace pytra::std::typing
